aries_cloudagent/askar/tests/test_store.py

```diff
@@ -0,0 +1,31 @@
+from aries_cloudagent.core.error import ProfileError
+from asynctest import TestCase as AsyncTestCase
+
+from ..store import AskarStoreConfig
+
+class TestStoreConfig(AsyncTestCase):
+    async def test_init_success(self):
+        config = {
+            "key_derivation_method": "RAW",
```

Comment:
"Raw" would be better, to also test the .lower() part.

Reply:
changed and moved to variable